.github/actions/dart_package/action.yaml

```diff
@@ -67,10 +67,18 @@ runs:
         dart pub global activate coverage
         dart test -j ${{inputs.concurrency}} --coverage=coverage --platform=${{inputs.platform}} && dart pub global run coverage:format_coverage --lcov --in=coverage --out=coverage/lcov.info --packages=.dart_tool/package_config.json --report-on=${{inputs.report_on}} --check-ignore
 
+    # Per https://stackoverflow.com/questions/73402042/github-action-expressions-split-string
+    - name: Split package name
+      env:
+        PACKAGE_PATH: ${{ github.ref_name }}
+      id: split
+      shell: ${{ inputs.shell }}
+      run: echo "::set-output name=package_name::${PACKAGE_PATH##*/}"
```

Comment:
I think set-output is deprecated
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
Fixed.